test-utils/src/main/kotlin/Extensions.kt

```diff
@@ -41,3 +46,23 @@ fun FileArchiver.Companion.createDefault(): FileArchiver =
         patterns = LicenseFilenamePatterns.DEFAULT.allLicenseFilenames.map { "**/$it" },
         storage = LocalFileStorage(DEFAULT_ARCHIVE_DIR)
     )
+
+fun TestConfiguration.createOrtTempDir(vararg infixes: String): File {
```

Comment:
Just a bit unsure whether the test functions should be named exactly like the ones used for production. Could there be a chance to mix this up?

Reply:
> Could there be a chance to mix this up?

Yes.

What name would you like better? `createTestTempDir` (so just replace "Ort" with "Test")?

Reply:
Yes, this name would be fine with me.